.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
 script:
   - coala --non-interactive -V
   - npm run build
+  - ./.ci/gather_data.sh
```

Comment:
could this be JavaScript, so it can be incorporated into the npm build process ?

Reply:
I currently have it incorporated in the build script when `npm run gather` is run. Would it still be preferable to have it in JavaScript?

Reply:
Yes, javascript, so it can be moved and re-used